mlops/common/pipeline_job_config.py

```diff
@@ -0,0 +1,38 @@
+"""A module to define the Pipeline class for setting up and managing pipeline job properties."""
+
+
+class PipelineJobConfig:
+    """A class to set up the common properties of a pipeline job."""
+
+    def __init__(
+        self,
+        environment_name: str,
+        build_reference: str,
+        published_model_name: str,
+        dataset_name: str,
+        build_environment: str,
+        wait_for_completion: str,
```

Comment:
should this be type bool?

Reply:
Please refer to my message on the similar comment